src/main/java/com/capitalone/dashboard/client/RestClient.java

```diff
@@ -33,107 +35,107 @@ public RestClient(Supplier<RestOperations> restOperationsSupplier) {
         this.restOperations = restOperationsSupplier.get();
     }
 
-    public ResponseEntity<String> makeRestCallPost(String url, JSONObject body) {
-        if (restOperations == null) { return null; }
+    private ResponseEntity<String> makeRestCallPost(String url, HttpHeaders headers, JSONObject body) {
```

Comment:
do we really need separate methods `makeRestCallPost` and `makeRestCallGet`? `restOperations.exchange()` signature is similar for both GET and POST. Since we are always converting to String.

Reply:
Look at them, there seem to be enough differences to loose the advantage of combining the two. One needs body the other does not, one use HttpHeaders other uses HttpEntity (cloned from the original code), plus method and op, there will be at least 4 conditional statements, given the size of the method, not enough benefits.